### ISSUE TITLE
Need slider with multiple handles

### ISSUE THREAD

**Issue by cont-kolomeets** [Esri/calcite-components#6645]
### Description

We need the ability to specify multiple values on a slider. Suggest to use a new `values` property that would accept and array of numbers.

### Acceptance Criteria

Provide a way to specify multiple values on a slider to show multiple handles.

### Which Component

https://developers.arcgis.com/calcite-design-system/components/slider/

### Example Use Case

Here is the result we are trying to achieve. The histogram will be drawn on our side, though.

![image](https://user-images.githubusercontent.com/5318527/227491838-ff8e35bb-56b5-4b3d-aa2e-27b5a8d96506.png)

### Priority impact

p2 - want for current milestone

### Esri team

ArcGIS Business/Community Analyst

**Comment by cont-kolomeets** [comment]
@macandcheese Yes, would be great to define the look of the histogram by specifying `histogramType = "bars" | "dots" | "interpolate" | "something else"`

**Comment by geospatialem** [comment]
Adding a spike to the above to kickstart the initial conversations internally with Calcite team members.

**Comment by DitwanP** [comment]
There was a newer issue (#11979) that was a duplicate of this one that got design sign off from Aaron [here](https://github.com/Esri/calcite-design-system/issues/11979#issuecomment-2884990637) 

> The current slider design can support [multiple thumbs](https://codepen.io/ashetland/pen/wBBOgdW). Adding more stops should not require any design changes at this time.

That issue was closed as a duplicate of this one.
